3rdparty/librealsense/librealsense.cmake

```diff
@@ -17,6 +17,9 @@ ExternalProject_Add(
     COMMAND ${GIT_EXECUTABLE} init
     COMMAND ${GIT_EXECUTABLE} apply --ignore-space-change --ignore-whitespace
         ${CMAKE_CURRENT_LIST_DIR}/fix-cudacrt.patch
+    # Patch for macOS ARM64 support for versions < 2.50.0
```

Comment:
TODO: Upgrade librealsense to latest so that the patch isn't needed. We can create an issue for this.

Reply:
I've opened #6714, however builds fail across all platforms currently due to linking and configuration errors. I'll check whats needed to fix them.